src/main/java/bio/terra/service/dataset/flight/inheritsteward/EnableInheritStewardFlight.java

```diff
@@ -43,5 +47,13 @@ public EnableInheritStewardFlight(FlightMap inputParameters, Object applicationC
     addStep(new GetSnapshotGoogleProjectIdsStep(snapshotService, datasetId));
     addStep(new SetAuthBqJobUserStep(resourceService, custodianEmail, true));
     addStep(new UnlockDatasetStep(datasetService, false));
+    addStep(
+        new JournalRecordUpdateEntryStep(
+            journalService,
+            userReq,
+            datasetId,
+            IamResourceType.DATASET,
+            "Enable inherit steward",
+            true));
```

Comment:
It looks like all uses of this journal step except for dataset and snapshot lock flights don't log the flight ID. How did you decide to log the flight ID in this case?